<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modbusconf</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Devices</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="listWidget_devices"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="toolButton_cloneDevice">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Clone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_addDevice">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_delDevice">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Active Device Settings</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_deviceName"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Slave ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_deviceSlaveID"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Discrete Inputs Start</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_deviceDiscreteInputsStart"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Discrete Inputs Size</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinBox_deviceDiscreteInputsSize"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Coils Start</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="spinBox_deviceCoilsStart"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Coils Size</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QSpinBox" name="spinBox_deviceCoilsSize"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Input Registers Start</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="spinBox_deviceInputRegistersStart"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Input Registers Size</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QSpinBox" name="spinBox_deviceInputRegistersSize"/>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Holding Registers Start</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QSpinBox" name="spinBox_deviceHoldingRegistersStart"/>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Holding Registers Size</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QSpinBox" name="spinBox_deviceHoldingRegistersSize"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_deviceAddress"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>IP Port</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_deviceIpPort"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Protocol</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_deviceProtocol"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>RTU Baud Rate</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinBox_deviceRtuBaudRate"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>RTU Parity</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_deviceRtuParity"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>RTU Data Bits</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QSpinBox" name="spinBox_deviceRtuDataBits"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>RTU Stop Bits</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="spinBox_deviceRtuStopBits"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>View</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
